### DISCUSSION TITLE
Failed to compile with create-react-app

### DISCUSSION THREAD

**Discussion by mesrbn** [verdaccio/verdaccio#2071] [Q&A] [answered]
Is there any solution that I will be able to use verdaccio with create-react-app out-of-the-box webpack?

**Comment by sergiohgz** [comment, marked as answer]
Sorry, but I cannot understand your question.

Can you provide a more detailed description of your needs? Without it, we cannot provide a proper answer

**Reply by mesrbn** [reply]
I solved the issue. Thanks a lot. The problem was not Verdaccio but create-react-app restrictions on customized customization.
